docs/usage.md

```diff
@@ -47,7 +47,7 @@ You can avoid running any startup scripts by adding `class="mejs__player"` to th
 ### Vanilla JavaScript
 ```html
 <script>
-	var player = new MediaElementPlayer('player', {
+	var player = new MediaElementPlayer(document.querySelector('#mediaplayer'), {
```

Comment:
Hmm, okay. I've tried the following setup bit it did not work for me:
```html
<video class=".js-mediaelement">…</video>
<script>
   new MediaElementPlayer('.js-mediaelement');
</script>
```

I had to use `document.querySelector()` to get it working.